setup.py

```diff
@@ -97,7 +97,7 @@ def get_extras_require() -> Dict[str, List[str]]:
             "nbval",
             "scikit-image",
             "scikit-learn>=0.19.0,<0.23.0",  # optuna/visualization/param_importances.py.
-            "xgboost<1.3",
+            "xgboost",
```

Comment:
Thanks for this. I didn't expect our CI failure... (I should've been more careful on the integration libraries updates)